thirdparty_containers/tomcat/build.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<project name="tomcat-test" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		Build tomcat-test Docker image
+	</description>
+
+	<!-- set properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/thirdparty_containers/tomcat-test" />
+	<property name="dist" location="${DEST}/${JAVA_VERSION}" />
```

Comment:
With new TestKitGen we don't need the "dist" for different JAVA_VERSION any more. 'DEST' is enough.

Reply:
@sophia-guo - so what specific change are you asking for @chandrams to make?  Are you asking to remove line 10, and any references elsewhere to dist?

Reply:
Yes, line 10 can be removed. Correspondingly line 15 should be changed